ush/config_defaults.yaml

```diff
@@ -653,7 +653,7 @@ workflow:
   #
   #-----------------------------------------------------------------------
   #
-  FIXdir: '{{ EXPTDIR if workflow_switches.RUN_TASK_MAKE_GRID else [user.HOMEdir, "fix"]|path_join }}'
+  FIXdir: '{{ EXPTDIR }}'
```

Comment:
NCO requires that fix files be placed under `$HOMEdir/fix` which is why we place the fix files there, and in EXPTDIR in "community" mode. The logic above is not straightforward since we are testing "RUN_TASK_MAKE_GRID", which is always set to false in NCO mode.

Reply:
@danielabdi-noaa Ok, I didn't realize NCO requires the fix files to be in the directory of the clone.  In that case, why not test the value of `RUN_ENVIR` instead of `RUN_TASK_MAKE_GRID`, since many researchers (i.e. non-NCO users) will be running in community mode but will turn off the `make_grid` task once their grid is all set up.  So the line would be changed to:
```
FIXdir: '{{ EXPTDIR if user.RUN_ENVIR == "community" else [user.HOMEdir, "fix"]|path_join }}'
```
I tested this with both `RUN_ENVIR = "nco"` and `RUN_ENVIR = "community"` and it works as expected in each case.  Are you ok with that solution?

Reply:
@gsketefian  Sounds good. Note that this logic came about when de-tangling NCO mode from the "grid being pre-generated" so that we can run all WE2E tests with either nco or community modes. It was easier to base many of the logic based on RUN_TASK_MAKE_GRID, but some logic like symlinking/copying fix files have since became their own options, while NCO mode always symlinked fix files and community mode copied. It is now actually symlink by default to save space on experiment directory size.

Reply:
@danielabdi-noaa Ok this one is updated now to check `RUN_ENVIR` instead of `RUN_TASK_MAKE_GRID`.